packages/prompts/README.md

```diff
@@ -339,6 +339,26 @@ import { confirm } from '@inquirer/prompts';
 const answer = await confirm({ message: await getMessage() });
 ```
 
+## Usage with `npx`
+
+You can use Inquirer prompts directly in the shell via [`npx`](https://docs.npmjs.com/cli/v8/commands/npx), which is useful for quick scripts or `package.json` commands.
+
+A community library, [@inquirer-cli](https://github.com/fishballapp/inquirer-cli), exposes each prompt as a standalone CLI.
+
+For example, to prompt for input:
+
+```bash
+$ npx @inquirer-cli/input 'Enter your name'
```

Comment:
```suggestion
name=$(npx @inquirer-cli/input -r "What is your name?")
echo "Hello, $name!"
```

That way we have a complete example! Should it uses `-y` to skip the install prompt?

Reply:
updated with `-y` too.